packages/enrichPrescriptions/src/enrichPrescriptions.ts

```diff
@@ -0,0 +1,44 @@
+import {Logger} from "@aws-lambda-powertools/logger"
+import {injectLambdaContext} from "@aws-lambda-powertools/logger/middleware"
+import {LogLevel} from "@aws-lambda-powertools/logger/types"
+import middy from "@middy/core"
+import inputOutputLogger from "@middy/input-output-logger"
+import {Bundle} from "fhir/r4"
+import {StatusUpdates, applyStatusUpdates} from "./statusUpdates"
+import {lambdaResponse} from "./responses"
+
+export const LOG_LEVEL = process.env.LOG_LEVEL as LogLevel
+const logger = new Logger({serviceName: "enrichPrescriptions", logLevel: LOG_LEVEL})
+
+export type EnrichPrescriptionsEvent = {
+  fhir: Bundle,
+  StatusUpdates?: {Payload: StatusUpdates}
+}
+
+export async function lambdaHandler(event: EnrichPrescriptionsEvent) {
+  const searchsetBundle = event.fhir
```

Comment:
should have logger.appendKeys near the start similar to what we have in getMyPrescriptions so that we log ids we can correlate to logs

Reply:
Added.